codalab/model/bundle_model.py

```diff
@@ -30,6 +30,8 @@
     bundle as cl_bundle,
     bundle_dependency as cl_bundle_dependency,
     bundle_metadata as cl_bundle_metadata,
+    bundle_store as cl_bundle_store,
+    bundle_location as cl_bundle_location,
     group as cl_group,
```

Comment:
Add tests for the new endpoints in a file similar to https://github.com/codalab/codalab-worksheets/blob/bundle-store-rest-endpoints/tests/unit/rest/bundles_test.py#L6